src/Psalm/Internal/Analyzer/ScopeAnalyzer.php

```diff
@@ -104,7 +104,6 @@ public static function getControlActions(
             if ($stmt instanceof PhpParser\Node\Stmt\Expression) {
                 if ($stmt->expr instanceof PhpParser\Node\Expr\FuncCall
                     && $stmt->expr->name instanceof PhpParser\Node\Name
-                    && $stmt->expr->name->parts === ['trigger_error']
                     && isset($stmt->expr->args[1])
                     && $stmt->expr->args[1]->value instanceof PhpParser\Node\Expr\ConstFetch
                     && in_array(
```

Comment:
The whole `if` doesn't make sense without that line, and needs to be dropped. You don't want to enter that `if` for every call to any function where second argument is one of `E_*` constants.

Reply:
Right, I made this change too quickly. I'll fix that